lib/vsc/mympirun/option.py

```diff
@@ -94,7 +94,9 @@ def make_init(self):
                        "str", "store", None),
 
             "output-check-timeout": ("Set timeout threshold to print warning if no output has been produced yet (sec)",
-                                     "int", "store", 600),
+                                     "int", "store", 3600),
+
+            "output-check-fatal": ("Set output check to be fatal after a set amount of time", None, "store_true", True),
```

Comment:
or `Exit with exitcode %s instead of warn if --output-check-timout is enabled` (well, `--output-check-timeout` is always enabled, but when non-negative).